intellij/src/nl/tudelft/watchdog/intellij/logic/ui/listeners/IntelliJMarkupModelListener.java

```diff
@@ -0,0 +1,15 @@
+package nl.tudelft.watchdog.intellij.logic.ui.listeners;
```

Comment:
This seems to be a first step toward implementing ASAT tracking functionality? Different PR?